.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 .vscode
-.obsidian
\ No newline at end of file
+.obsidian
+.env
+Makefile
```

Comment:
makefiles are usually part of the dev process, fine to make on and leave it in the repo as the sole version